common/app/templates/inlineJS/blocking/applyAdfreeRenderCondition.scala.js

```diff
@@ -0,0 +1,119 @@
+@()
+
+(function applyAdfreeRenderCondition(commercial){
+    var AD_FREE_COOKIE = 'gu_adfree_test',
+        MVT_ID_COOKIE = 'GU_mvt_id',
+        FIRST_TIME_VISIT_KEY = 'gu.firstVisitTime',
+        ALREADY_VISITED_KEY = 'gu.alreadyVisited';
+
+    if ((readLocalStorage(ALREADY_VISITED_KEY) || 0) == 0) {
+        writeLocalStorage(FIRST_TIME_VISIT_KEY, new Date().getTime());
+    }
+
+    if (isFirst3VisitDays() && isInVariantGroup()){
+        commercial.showingAdfree = true;
+        return 'is-adfree';
+    } else {
+        commercial.showingAdfree = false;
+        return '';
+    }
+
+    function isFirst3VisitDays() {
+        var firstVisit = readLocalStorage('gu.firstVisitTime') || 0;
```

Comment:
Thanks, I'll fix the constant.

The gu.firstVisitTime is a new cookie, so won't exist for non-new users.  This test is only interested in users who are new so using unix epoch instead of unknown is fine.